src/main/java/frc/team88/swerve/util/Vector2D.java

```diff
@@ -24,7 +24,7 @@
      * @param x
      * @param y
      */
-    private Vector2D(double x, double y) {
```

Comment:
The idea is to keep this private so that users are forced to use Vector2D.createCartesianCoordinates, otherwise it isn't obvious whether the input is cartesian or polar